src/components/visualizations/chart/highcharts/customConfiguration.ts

```diff
@@ -938,8 +941,8 @@ function getYAxisTickConfiguration(chartOptions: any, axisPropsKey: string) {
     };
 }
 
-function getAxesConfiguration(chartOptions: any) {
-    const { type } = chartOptions;
+function getAxesConfiguration(chartOptions: IChartOptions) {
+    const { type }: any = chartOptions;
```

Comment:
This doesn't look right. chartOptions is IChartOptions strict null checks are not set. So `: any` removes type information here. Make sure you use TS version from the workspace instead of the global one from say VS code.

Reply:
Since `type` is passed to `function getAxisLineConfiguration(chartType: ChartType, isAxisVisible: boolean) {}`.
The problem is `type` in `IChartOptions` is string and can not pass to `getAxisLineConfiguration` as `ChartType`.

My new solution is `const type = chartOptions.type as ChartType;`
What do you think? :)